From d7adb5cd23c4f6cefb9ac25891441e47299ed742 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 13 Dec 2017 15:27:57 +0500
Subject: [PATCH] configure.ac: fix the llvm version correctly

We do not use the LLVM patch version in OE so
drop that from the version string as well.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index d132523..12dedd6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1032,7 +1032,7 @@ strip_unwanted_llvm_flags() {
 
 llvm_set_environment_variables() {
     if test "x$LLVM_CONFIG" != xno; then
-        LLVM_VERSION=`$LLVM_CONFIG --version | cut -c1-5`
+        LLVM_VERSION=`$LLVM_CONFIG --version | cut -c1-3`
         LLVM_CPPFLAGS=`strip_unwanted_llvm_flags "$LLVM_CONFIG --cppflags"`
         LLVM_INCLUDEDIR=`$LLVM_CONFIG --includedir`
         LLVM_LIBDIR=${llvm_prefix}
@@ -2683,7 +2683,7 @@ if test "x$enable_llvm" = xyes; then
     dnl (See https://llvm.org/bugs/show_bug.cgi?id=6823)
     if test "x$enable_llvm_shared_libs" = xyes; then
         dnl We can't use $LLVM_VERSION because it has 'svn' stripped out,
-        LLVM_SO_NAME=LLVM-`$LLVM_CONFIG --version|cut -c1-5`
+        LLVM_SO_NAME=LLVM-`$LLVM_CONFIG --version|cut -c1-3`
         AS_IF([test -f "$LLVM_LIBDIR/lib$LLVM_SO_NAME.$IMP_LIB_EXT"], [llvm_have_one_so=yes])
 
         if test "x$llvm_have_one_so" = xyes; then
-- 
2.11.1

